examples/anonymize_dataset.py

```diff
@@ -203,6 +203,11 @@
 # in a dataset with multiple subjects will the effects of randomly-picked IDs
 # become apparent.)
 #
+# A good random seed is truly random. Avoid using random seeds from popular
+# culture, like "42", or "1337". To obtain a truly random seed, you can paste
+# the following into your console:
+# ``python -c "import secrets; print(secrets.randbits(31))"``
```

Comment:
do you think it's worth explaining where `31` comes from here? Or is that too much information / too distracting from the main point?